src/common/PolarStereographicProjection.cc

```diff
@@ -428,7 +428,7 @@ void PolarStereographicProjection::gridLongitudes(const GridPlotting& grid)  con
 	{
 		if (*lon < xmin_ ) continue;
 		if (*lon > xmax_ ) continue;
-		Polyline poly;
+		magics::Polyline poly;
```

Comment:
Same thing here, why did you have to specify the namespace ?

Reply:
Same as the others - because of the clash with the 'Polyline' function defined in windows.h

Reply:
I have my answer now